### ISSUE TITLE
problems about training on a single card

### ISSUE THREAD

**Issue by Utopin** [GuangxingHan/Meta-Faster-R-CNN#25]
Hi, thangs for your work. I have a 16gb gpu and in order to run the training process, i change the support_shot from 30 to 20 and the batch size from 8 to 2.But i don't know whether it is better to reduce the support_shot more and increase the batch size. Apart from that, do i need to chage the learning weight and iteration nums? I'd be appreciated if you can give me some help!

**Comment by GuangxingHan** [comment]
Hi, I am afraid the accuracy of the model would be affected. Both of the training support_shot and batch size are crucial, and I cannot tell which one is more important. You can also decrease the learning rate and increase the iteration number if you have time for a try.
